### PR TITLE
Update thread template

### PR DESCRIPTION
**Pull request by desudesutalk**
Duplicate links for "return" and "go to top removed".

Added width reset for `div.pages`, this should help mobile users.

### CODE REVIEW

File: templates/thread.html [hunk 1]
```diff
@@ -67,9 +65,9 @@
 	<div class="bar bottom">
 		<div class="threadlinks">
 			<span class="threadlink">[ <a href="{{ return }}">{% trans %}Return{% endtrans %}</a> /</span>
-        		<span class="threadlink"><a href="#" style="padding-left: 10px"> {% trans %}Go to top{% endtrans %}</a> /</span>
+        		<span class="threadlink"><a href="#" id="bottom" style="padding-left: 10px"> {% trans %}Go to top{% endtrans %}</a> /</span>
 		</div>
-		<div class=pages></div>
+		<div class=pages style="width: initial;"></div>
```
Comment (barrucadu) [inline templates/thread.html]: I think this should go in the relevant CSS file, rather than inline.

Comment (desudesutalk) [inline templates/thread.html]: Nope. It already has width there
https://github.com/lainchan/lainchan/blob/master/stylesheets/style.css#L487

This one is a fix for case of thread view (where is no page links) and is specially for mobiles(small screen sizes)

Comment (desudesutalk) [inline templates/thread.html]: But now i think what width for `div.pages` can be completely removed. Should not break anything.

Can't get why it was set initially.

**Comment by desudesutalk** [comment]
Correct bottom link. Must be at the bottom of page and not in bar for scrolling to work.

**Pull request merged**